velox/docs/functions/spark/string.rst

```diff
@@ -61,6 +61,13 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT endswith('js SQL', 'js'); -- false
         SELECT endswith('js SQL', NULL); -- NULL
 
+.. spark:function:: empty2null(input) -> varchar
+
+    Returns null when the input is empty, otherwise, it returns the input itself. ::
```

Comment:
nit: capitalize null, and use \``input\`` to refer to the variable.

Document suggestion:
> Returns NULL if \``input\`` is empty. Otherwise, returns \``input\``.

Reply:
+1 @rui-mo's suggestion. @Yohahaha, could you check this comment? Thanks!

Reply:
ok